helm/sematic-server/templates/configmap.yaml

```diff
@@ -25,6 +25,6 @@ data:
 {{ if .Values.server.sematic_dashboard_url }}
   SEMATIC_DASHBOARD_URL: {{ .Values.server.sematic_dashboard_url | quote }}
 {{ end }}
-{{ if .Values.sematic.socket_io_address }}
-  SEMATIC_SOCKET_IO_ADDRESS: {{ .Values.sematic.socket_io_address }}
+{{ if .Values.deployment.socket_io.dedicated }}
```

Comment:
I think it would be preferable to *always* have a dedicated socket io instance. If we allow too much configuration (some deployments have a dedicated instance, others don't), then it will be harder for us to do exhaustive testing and debug user setups.

Reply:
this isn't intended to be a long term solution for us, since we want to implement a proper message queue to back the socketio instance, so i wouldn't want to propagate this hack widely if we can avoid it.  once the message queue is in place, i would agree with you.